common/changes/office-ui-fabric-react/choicegroup_2019-01-04-02-15.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "expose focus() method in IChoiceGroup",
```

Comment:
nit: this comment gets public exposure in the release notes. Can you please format it like so: _'ChoiceGroup: exposing focus() method in IChoiceGroup and makes sure it is implemented.'_? Makes it much easier to scan through the release notes and looks more like an official release documentation. Thanks!